src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,53 @@
+@keyframes rotate {
+  from {
+    transform: rotate(0);
+  }
+  to {
+    transform: rotate(1turn);
+  }
+}
+
+.stopwatch {
+  width: $clock-size;
```

Comment:
create mixin for creating of circle, you can use it twice, here and for pseudoelement